src/utils/sanitization.ts

```
@@ -0,0 +1,197 @@
+ * Validates and sanitizes string input for SQL queries
+ * Removes dangerous characters and limits length
+ * 
+ * @param input - The input string to sanitize
+ * @param maxLength - Maximum allowed length
+ * @returns Sanitized string safe for SQL queries
+ */
+export function sanitizeString(input: string, maxLength: number = 255): string {
+  if (typeof input !== 'string') return '';
+  
+  return input
+    .replace(/[\x00-\x1F\x7F-\x9F]/g, '') // Remove control characters
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)